scripts/exregional_run_observer.sh

```diff
@@ -0,0 +1,545 @@
+#
+print_info_msg "
+========================================================================
+Entering script:  \"${scrfunc_fn}\"
+In directory:     \"${scrfunc_dir}\"
+
+This is the ex-script for the task that runs a analysis with FV3 for the
+specified cycle.
+========================================================================"
+#
+#-----------------------------------------------------------------------
+#
+# Specify the set of valid argument names for this script/function.  
+# Then process the arguments provided to this script/function (which 
+# should consist of a set of name-value pairs of the form arg1="value1",
+# etc).
+#
+#-----------------------------------------------------------------------
+#
+valid_args=( "CYCLE_DIR" "ANALWORKDIR" "FG_ROOT")
+process_args valid_args "$@"
+#
+#-----------------------------------------------------------------------
+#
+# For debugging purposes, print out values of arguments passed to this
+# script.  Note that these will be printed out only if VERBOSE is set to
+# TRUE.
+#
+#-----------------------------------------------------------------------
+#
+print_input_args valid_args
+#
+#-----------------------------------------------------------------------
+#
+# Load modules.
+#
+#-----------------------------------------------------------------------
+#
+case $MACHINE in
+#
+"WCOSS_C" | "WCOSS")
+#
+
+  if [ "${USE_CCPP}" = "TRUE" ]; then
+  
+# Needed to change to the experiment directory because the module files
+# for the CCPP-enabled version of FV3 have been copied to there.
+
+    cd_vrfy ${CYCLE_DIR}
+  
+    set +x
+    source ./module-setup.sh
+    module use $( pwd -P )
+    module load modules.fv3
+    module list
+    set -x
+  
+  else
+  
+    . /apps/lmod/lmod/init/sh
+    module purge
+    module use /scratch4/NCEPDEV/nems/noscrub/emc.nemspara/soft/modulefiles
+    module load intel/16.1.150 impi/5.1.1.109 netcdf/4.3.0 
+    module list
+  
+  fi
+
+  ulimit -s unlimited
+  ulimit -a
+  APRUN="mpirun -l -np ${PE_MEMBER01}"
+  ;;
+#
+"THEIA")
+#
+
+  if [ "${USE_CCPP}" = "TRUE" ]; then
+  
+# Need to change to the experiment directory to correctly load necessary 
+# modules for CCPP-version of FV3LAM in lines below
+    cd_vrfy ${EXPTDIR}
+  
+    set +x
+    source ./module-setup.sh
+    module use $( pwd -P )
+    module load modules.fv3
+    module load contrib wrap-mpi
+    module list
+    set -x
+  
+  else
+  
+    . /apps/lmod/lmod/init/sh
+    module purge
+    module use /scratch4/NCEPDEV/nems/noscrub/emc.nemspara/soft/modulefiles
+    module load intel/16.1.150 impi/5.1.1.109 netcdf/4.3.0 
+    module load contrib wrap-mpi 
+    module list
+  
+  fi
+
+  ulimit -s unlimited
+  ulimit -a
+  np=${SLURM_NTASKS}
+  APRUN="mpirun -np ${np}"
+  ;;
+#
+"HERA")
+  ulimit -s unlimited
+  ulimit -a
+  APRUN="srun"
+  LD_LIBRARY_PATH="${UFS_WTHR_MDL_DIR}/FV3/ccpp/lib${LD_LIBRARY_PATH:+:$LD_LIBRARY_PATH}"
+  ;;
+#
+"JET")
+  ulimit -s unlimited
+  ulimit -a
+  APRUN="srun"
```

Comment:
The latest exregional_run_analysis.sh added the following two lines to speed up GSI run time
  export OMP_NUM_THREADS=2
  export OMP_STACKSIZE=1024M
GSI_observer may run much faster and don't need this.

Reply:
Probably so. These two lines might be added later if needed.

Reply:
Added the above 2 lines to exregional_run_observer.sh. Thanks!